packages/iris-grid/src/IrisGridRenderer.ts

```diff
@@ -187,12 +187,10 @@ class IrisGridRenderer extends GridRenderer {
       return;
     }
 
-    const lastGroupedColumn = groupedColumns[groupedColumns.length - 1];
-    const modelIndex = columns.findIndex(
-      c => c.name === lastGroupedColumn.name
-    );
-    const columnX = allColumnXs.get(modelIndex);
-    const columnWidth = allColumnWidths.get(modelIndex);
+    const lastVisibleGroupColumnIndex = groupedColumns.length - 1;
```

Comment:
I would add a comment along the lines of "This assumes the engine puts the grouped columns at the start of the model, so model and visible index match for grouped columns. If we add freeze support for tree tables or allow moving the grouped columns, this may need to change to get the visible index from the model index" so anybody woh sees this code in the future knows we didn't get the visible from model on purpose

Reply:
that sounds like its waiting to break @mofojed might have opinions.

Reply:
It was already broken before if we added support for either of the things I mentioned. `allColumnXs` is keyed on `VisibleIndex` and we were using `ModelIndex`

Reply:
@mattrunyon I've added a comment. It also occurs to me if we do ever support group columns not being the furthest to the left, we'll have some other UI decisions to make since the single vertical line of separation would no longer convey what it does now.

Reply:
Frozen columns get a double line and a background.

![image](https://user-images.githubusercontent.com/1576283/223453126-5e4e9667-1094-4de4-b4d0-74557d2e3b31.png)

We should probably not crash the table when a frozen column and a rollup are combined, what behaviour happens though I am not sure yet.

Reply:
@dsmmcken  There's definitely some issues with combining frozen + grouping. A couple of example:

- freeze column then group. Frozen columns no longer highlighted and context menu says you can freeze it. If you remove the grouping, frozen column returns.
- Freeze a grouped column causes an error

That said, I don't think these are caused by this PR and likely will require some more testing + requirement definition of how we want to handle various scenarios of grouping + freeze + unfreeze

Any objections to this PR getting merged and us creating a new issue for the above?

Reply:
I'm fine with it. I was waiting to see if @mofojed had anything to say in regards to this, but the existing behavior would break if either of those features were implemented anyway, so I think it's fine.

Just for future reference, we typically click the re-request review button next to the reviewers when we're ready for a re-review. That makes the PR show up as "awaiting your review" in VSCode or when filtering the PR list by PRs waiting on your review